src/OrleansAzureUtils/Storage/Exceptions.cs

```diff
@@ -48,7 +48,7 @@ public class TableStorageUpdateConditionNotSatisfiedException : InconsistentStat
         /// <summary>
         /// Id of grain
         /// </summary>
-        public GrainReference GrainId { get; }
+        public string GrainId { get; }
```

Comment:
>public string GrainId { get; } [](start = 8, length = 30)

Why do we need to switch from `GrainReference` to `string` here? Seems unrelated to elimination of statics.